esrally/mechanic/telemetry.py

```diff
@@ -443,14 +443,19 @@ def record_stats_per_index(self, name, stats):
 
 
 class NodeStats(TelemetryDevice):
+    """
+    Gathers different node stats.
+    """
+
     internal = False
     command = "node-stats"
     human_name = "Node Stats"
     help = "Regularly samples node stats"
-
-    """
-    Gathers different node stats.
+    warning = """You have enabled the node-stats telemetry device, but requests to the _nodes/stats Elasticsearch endpoint
+          trigger additional refreshes and WILL SKEW results.
```

Comment:
Not sure if we need to upper-case?

Reply:
Just wanted to really ensure this is visible (because there may be other info/warn messages).